HangMan/src/hangman.rs

```diff
@@ -0,0 +1,139 @@
+use std::vec;
+use dioxus::prelude::*;
+
+//component for managing the image of the hangman
+#[inline_props]
+pub fn Hangman(cx:Scope, stage: usize, reset_image: bool) -> Element {
+    
+    let css_status = use_ref(cx, || {vec![true;11]});    
```

Comment:
Instead of `css_status`, it could be more clear to name this something like `body_parts_hidden`.

In normal handman, this can be represented with a single number for the number of body parts that have been drawn which similify this and let you remove the